XenModel/Utils/Helpers.cs

```diff
@@ -1934,5 +1934,11 @@ public static bool VLAN0Allowed(IXenConnection connection)
            // For Creedence or later on the vSwitch backend, we allow creation of VLAN 0
            return master != null && CreedenceOrGreater(master) && master.vSwitchNetworkBackend;
        }
+
+       public static bool ContainerCapability(IXenConnection connection)

```

Comment:
This looks good, but is it sufficient to check only for the plugin or do we need a Cream or greater check here as well? (Or are we sure this plugin cannot be installed on any pre-Cream host?)